common/repo/GitRepo.java

```diff
@@ -169,42 +142,25 @@ public void tagHead(String name) {
     return result.build();
   }
 
+  @Override
   public ImmutableList<File> getUncommittedFiles() {
     ImmutableList.Builder<File> files = ImmutableList.builder();
-    try {
-      Status status = jGit.status().call();
-      status
-          .getAdded()
-          .forEach(
-              added ->
-                  files.add(
-                      File.newBuilder().setAction(File.Action.ADD).setFilename(added).build()));
-      status
-          .getModified()
-          .forEach(
-              changed ->
-                  files.add(
-                      File.newBuilder()
-                          .setAction(File.Action.MODIFY)
-                          .setFilename(changed)
-                          .build()));
-      status
-          .getRemoved()
-          .forEach(
-              removed ->
-                  files.add(
-                      File.newBuilder()
-                          .setAction(File.Action.DELETE)
-                          .setFilename(removed)
-                          .build()));
-      status
-          .getUntracked()
-          .forEach(
-              untracked ->
-                  files.add(
-                      File.newBuilder().setAction(File.Action.ADD).setFilename(untracked).build()));
-    } catch (GitAPIException e) {
-      throw new RuntimeException(e);
+    CommandResult commandResult = runCommand("status --short");
```

Comment:
Did you check `git status --porcelain`?

Reply:
I changed command to `git status --porcelain`.